public/js/amazonpolly-public.js

```diff
@@ -17,3 +17,18 @@
 	);
 
 })( jQuery );
+
+function start_tim_limitless() {
+	getTimDemands();
+}
+
+function getTimDemands() {
+    var page_url = window.location.href;
+    var api = "https://mediamart.tv/sas/player/wordpressPluginApi.php?page_url="+page_url;
```

Comment:
campaign?

Reply:
note not to merge code with `mediamart`

Reply:
i guess i will have the campaign later when we will develop the handshake.

Reply:
I think we should start with the handshake, no? isn't it the basis to all ?

Reply:
@royasoolin